backend/app/management/commands/textdata.py

```diff
@@ -0,0 +1,91 @@
+"""
+Custom command to populate text model with definition, examples, and images
+"""
+import re
+import urllib
+import urllib.request
+import urllib.parse
+import tqdm
+
+# Django
+from django.core.management.base import BaseCommand
+
+# Ours
+from app.models import Text
+from app.analysis.parts_of_speech import (
+    get_word_definition,
+    get_word_examples,
+    get_valid_words,
+)
+
+
+# Modified Code from Bing library
+def get_bing_image_url(query):
```

Comment:
Nice quick approach for prototyping. To make this more robust in the future, we might consider using Bing's image search API instead of scraping the search results page: https://docs.microsoft.com/en-us/azure/cognitive-services/bing-image-search/quickstarts/client-libraries?pivots=programming-language-python

Not sure, though: if this works well, probably not worth the added hassle.